src/test/kotlin/com/yapp/giljob/domain/quest/dao/QuestRepositoryTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.yapp.giljob.domain.quest.dao
+
+import com.yapp.giljob.domain.position.domain.Position
+import com.yapp.giljob.domain.quest.domain.Quest
+import com.yapp.giljob.domain.quest.domain.QuestParticipation
+import com.yapp.giljob.domain.quest.domain.QuestParticipationPK
+import com.yapp.giljob.domain.subquest.domain.SubQuest
+import com.yapp.giljob.domain.user.dao.UserRepository
+import com.yapp.giljob.domain.user.domain.User
+import com.yapp.giljob.global.config.QuerydslTestConfig
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.context.annotation.Import
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Import(QuerydslTestConfig::class)
+class QuestRepositoryTest @Autowired constructor(
+    private val questRepository: QuestRepository,
+    private val questParticipationRepository: QuestParticipationRepository,
+    private val userRepository: UserRepository
+) {
+
+    private var userList = mutableListOf<User>()
+
+    private var questList = mutableListOf<Quest>()
+
+    @BeforeEach
+    fun setUp() {
+        for (i in 1..5L) {
+            userList.add(
+                userRepository.save(
+                    User(socialId = "testSocialId", nickname = "testNickname", position = Position.BACKEND)
+                )
+            )
+
+        }
```

Comment:
```suggestion
        repeat(5) {
            userList.add(
                userRepository.save(
                    User(socialId = "testSocialId", nickname = "testNickname", position = Position.BACKEND)
                )
            )
        }
```
index를 사용하지 않는다면 단순하게 repeat 를 사용할 수 있습니다 ㅎㅎ

Reply:
오! 감사합니다~~ㅎㅎㅎ